policyengine_us/variables/gov/states/va/tax/income/va_non_refundable_credits.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class va_non_refundable_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Virginia non-refundable credits"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.VA
```

Comment:
need to add the adds function: 
adds = "gov.states.va.tax.income.credits.non_refundable"